utils/nctl/sh/assets/setup.sh

```diff
@@ -133,6 +137,34 @@ validator_public_key = "${ACCOUNT_KEY_OF_VALIDATOR}"
 EOM
 }
 
+#######################################
+# Sets entry in chainspec's accounts.toml for a user account.
+# Arguments:
+#   Path to file containing a user's ed25519 public key in hex format.
+#   Path to file containing a validator's ed25519 public key in hex format.
```

Comment:
Is this correct?  Same question for the `Initial user staking weight.` 2 lines comment below.

Reply:
Thx. I will review the function comments plus wider documentation.